cypress/regression/CSS.spec.js

```diff
@@ -0,0 +1,13 @@
+describe('Matchbox CSS', () => {
+  it('renders correctly - desktop', () => {
+    cy.visit('/iframe.html?path=Visual-Regression__CSS');
```

Comment:
Percy isn't showing the right page for this, i think this is case sensitive `?path=Visual-Regression__Matchbox-Css`

Reply:
https://percy.io/efff6832/matchbox/builds/14582311/changed/824414572?browser=edge&browser_ids=18%2C19%2C20%2C21&subcategories=unreviewed%2Cchanges_requested&viewLayout=overlay&viewMode=new&width=1280&widths=375%2C1280